requirements.txt

```diff
@@ -11,3 +11,4 @@ phonenumbers==7.2.6
 django-cities-light==3.2.0
 Unidecode==0.4.19
 django-braces==1.9.0
+pycodestyle==2.3.1
```

Comment:
wouldn't `flake8` would do, do we need `pycodestyle` if we have `flake8`? 🤔

Reply:
Yes flake8 will do. I have removed pycodestyle.